cfme/tests/configure/test_timeprofile.py

```diff
@@ -2,82 +2,105 @@
+from cfme.configure.settings import TimeProfileEditView
 from cfme.utils.update import update
 
+
 pytestmark = [pytest.mark.tier(3),
               test_requirements.settings]
 
 
-def new_timeprofile():
-    return st.Timeprofile(description='time_profile' + fauxfactory.gen_alphanumeric(),
-                          scope='Current User',
-                          days=True,
-                          hours=True,
-                          timezone="(GMT-10:00) Hawaii")
-
-
 @pytest.mark.sauce
-def test_timeprofile_crud():
-    timeprofile = new_timeprofile()
-    timeprofile.create()
-    with update(timeprofile):
-        timeprofile.scope = 'All Users'
-    copied_timeprofile = timeprofile.copy()
-    copied_timeprofile.delete()
-    timeprofile.delete()
+def test_time_profile_crud(appliance):
+    """
+        This test case performs the CRUD operation.
+    """
+    collection = appliance.collections.time_profiles
+    time_profile = collection.create(
+        description='time_profile {}'.format(fauxfactory.gen_alphanumeric()),
+        scope='Current User',
+        days=True, hours=True,
+        timezone="(GMT-10:00) Hawaii")
+    with update(time_profile):
+        time_profile.scope = 'All Users'
+    collection.delete(False, time_profile)
 
 
 @pytest.mark.sauce
-def test_timeprofile_name_max_character_validation():
-    tp = st.Timeprofile(
+def test_time_profile_name_max_character_validation(appliance):
+    """
+    This test case performs the check for max character validation.
+    """
+    collection = appliance.collections.time_profiles
+    time_profile = collection.create(
         description=fauxfactory.gen_alphanumeric(50),
         scope='Current User',
+        days=True, hours=True,
         timezone="(GMT-10:00) Hawaii")
-    tp.create()
-    tp.delete()
+    collection.delete(False, time_profile)
 
 
 @pytest.mark.sauce
-def test_days_required_error_validation(soft_assert, appliance):
-    tp = st.Timeprofile(
-        description='time_profile' + fauxfactory.gen_alphanumeric(),
-        scope='Current User',
-        timezone="(GMT-10:00) Hawaii",
-        days=False,
-        hours=True)
-    tp.create(cancel=True)
-    view = appliance.browser.create_view(TimeProfileAddFormView)
-    soft_assert(view.timeprofile_form.help_block.text == "At least one day needs to be selected")
-    soft_assert(view.timeprofile_form.save_button.disabled)
-    view.timeprofile_form.cancel_button.click()
+def test_days_required_error_validation(appliance, soft_assert):
+    """
+    This test case performs the error validation of days field.
+    """
+    collection = appliance.collections.time_profiles
+    time_profile = collection.instantiate(description='UTC', scope='Current User',
+                                          days=True, hours=True)
+    with update(time_profile):
+        time_profile.days = False
+    view = appliance.browser.create_view(TimeProfileEditView)
```

Comment:
**QUESTION**
It's not clear to me how can you create `TimeProfileEditView` view if you already left the edit screen?

Reply:
Actually the save button is not active after we make changes with the help of update method and it's the reason because of which the save button is not clicked after performing the update operation and hence we are at the same page i.e edit page.
So I am creating the edit view over there.